### PR TITLE
fix: replaced switch camera text with symbol

### PR DESCRIPTION
**Pull request by shackerman**

### CODE REVIEW

File: examples/objectdetection/src/CameraStream.tsx [hunk 1]
```diff
@@ -132,7 +133,7 @@ export const CameraStream: React.FC<Props> = () => {
           ))}
         </Canvas>
         <Pressable style={styles.cameraSwitchButton} onPress={setActiveCamera}>
-          <Text style={styles.cameraSwitchButtonText}>Switch Camera</Text>
+          <Ionicons name="repeat-outline"></Ionicons>
```
Comment (cdiddy77) [inline examples/objectdetection/src/CameraStream.tsx:136]: please address the linter error by turning this into an empty tag.

**Review by cdiddy77 — APPROVED** [review]
Please address the one command and then once you have added that commit, ping me on the discord.

File: examples/objectdetection/src/CameraStream.tsx (hunk 1 above)
Comment (cdiddy77) [reply examples/objectdetection/src/CameraStream.tsx:136]: @shackerman see above --^

Comment (shackerman) [reply examples/objectdetection/src/CameraStream.tsx]: alright, fixed it!

**Pull request merged**